src/cloudant/database.py

```diff
@@ -17,34 +17,39 @@
 from .changes import Feed
 
 
-class CloudantDatabase(dict):
+class CouchDatabase(dict):
     """
-    _CloudantDatabase_
+    _CouchDatabase_
+
+    dict based interface to a CouchDB Database.
+    Instantiated with a reference to an account/session
+    it supports accessing the documents, and various database
+    features such as the indexes, changes feed, and design documents
+
+    :param account: CouchAccount instance corresponding to the db server
+    :param database_name: Name of the database
+    :param fetch_limit: Optional, sets the max number of docs to fetch per query
+        during iteration cycles
 
     """
-    def __init__(self, cloudant, database_name, fetch_limit=100):
-        super(CloudantDatabase, self).__init__()
-        self._cloudant_account = cloudant
-        self._database_host = cloudant._cloudant_url
+    def __init__(self, account, database_name, fetch_limit=100):
+        super(CouchDatabase, self).__init__()
+        self._cloudant_account = account
```

Comment:
figured I would get the refactor working first and then tweak the variable names after, since a S&R with working tests is always easier